rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -367,42 +367,9 @@ namespace rocal{
         m.def("rocalCopyToOutputTensor16",&wrapper_tensor16);
         // rocal_api_data_loaders.h
         m.def("COCO_ImageDecoderSlice",&rocalJpegCOCOFileSourcePartial,"Reads file from the source given and decodes it according to the policy",
-            py::return_value_policy::reference,
```

Comment:
We removed all the params and retained only py::return_value_policy::reference.  Is this correct?

Reply:
Yes, this is just a way of passing args in pybind, This is correct